server/src/gameMachine/gameReducer.ts

```diff
@@ -347,6 +322,33 @@ export function createGameReducer(initialState?: BaseUnifiedGame) {
             return state;
         }
       },
+      reminders: (state = [], action) => {
+        switch (action.type) {
+          case "KickPlayer":
+            return state.filter(
+              (reminder) =>
+                reminder.fromPlayer !== action.player &&
+                reminder.toPlayer !== action.player,
+            );
+          case "AddPlayerReminder":
+            return [...state, action.reminder];
+          case "ClearPlayerReminder": {
+            const reminder = state.find(
+              (reminder) => reminder.id === action.reminderId,
+            );
+            return reminder
```

Comment:
instead lets keep these in the same order:
`state.map(item => item.id === reminder?.id ? {...item, active: false} : item)`